.CI/create_feedstocks.py

```diff
@@ -264,18 +264,27 @@ def write_token(name, token):
         # Finish quietly.
         pass
 
+    # Parse `git status --porcelain` to handle some merge conflicts and generate the removed recipe list.
+    changed_files = subprocess.check_output(['git', 'status', '--porcelain', recipe_directory_name],
+                                            universal_newlines=True)
+    changed_files = changed_files.splitlines()
+
+    # Add all files from AU conflicts. They are new files that we weren't tracking previously.
+    # Adding them resolves the conflict and doesn't actually add anything to the index.
+    new_file_conflicts = filter(lambda _: _.startswith("AU "), changed_files)
+    new_file_conflicts = map(lambda _ : _.replace("AU", "", 1).lstrip(), new_file_conflicts)
+    for each_new_file in new_file_conflicts:
+        subprocess.check_call(['git', 'add', each_new_file])
```

Comment:
This is the heart of the change. Basically we just add the `AU` conflict files. This clears the conflict from the index, but does not effect what we will commit.